palette/src/blend/blend.rs

```diff
@@ -32,22 +32,22 @@ where
     ///acceptable, including functions and closures.
     ///
     ///```
-    ///use palette::{LinSrgb, LinSrgba, Blend};
-    ///use palette::blend::PreAlpha;
+    /// use palette::{LinSrgb, LinSrgba, Blend};
```

Comment:
Shouldn't the space at the beginning be applied for the whole comment block? That's how it is for newer files. I see you made similar changes elsewhere.